V5 Release note - Connectivity part (#876)
akeneo/pim-helpcenter: content/updates/connectivity/3-Synchronize-your-product-catalog-more-easily.md

```json
{
    "collection": [
        {
            "locale": null,
            "scope": null,
            "data": [
                "spring_2020",
                "summer_2020"
            ]
        }
    ]
}
```

Now, it receives codes and labels:

```json
{
    "collection": [
        {
            "locale": null,
            "scope": null,
            "data": [
                "spring_2020",
                "summer_2020"
            ],
            "linked_data": {
                "spring_2020": {
                    "attribute": "collection",
                    "code": "spring_2020",
                    "labels": {
                        "de_DE": "Frühling 2020",
                        "en_US": "Spring 2020",
                        "fr_FR": "Printemps 2020"
                    }
                },
                "summer_2020": {
                    "attribute": "collection",
                    "code": "summer_2020",
                    "labels": {
                        "de_DE": "Sommer 2020",
                        "en_US": "Summer 2020",
                        "fr_FR": "Été 2020"
                    }
                }
            }
        }
    ]
}
```

> In this example, we avoid 5 API calls per product. Can you imagine the impact for a catalog which contains thousands of products?

Reviewer: ```suggestion
In this example, we avoid 5 API calls per product. Can you imagine the impact on a catalog that contains thousands of products?
```

## New filters on categories, attributes, families and attribute groups
::: meta-data type="Improvement" ee-only="false" link-to-doc="TODO"
:::

> We introduced a new `search` parameter when retrieving attributes, families, categories or attribute groups. It will allow you to add filters to your requests that will help reduce the amount of data to process and the size of the API call responses.

Reviewer: ```suggestion
We introduced a new `search` parameter when retrieving attributes, families, categories, or attribute groups. It allows you to add filters to your requests which will help reduce the amount of data to process and the size of the API call responses.
```

Here are the available filters:
- **code**: allows you to filter on specific codes,
- **updated date**: allows you to retrieve only data updated since a specific date and hour (only available for attributes, families and attribute groups),
- **type**: allows you to filter on attributes from a specific type (only available for attributes),
- **parent**: allows you to retrieve all sub-categories of a parent category (only available for categories).

> You can combine filters to achieve powerful results!

Reviewer: ```suggestion
You can combine filters for more powerful results!
```

Let's take an example: you have an eCommerce platform that uses your PIM data. You know the list of all the attributes you need, and your application synchronizes the catalog structure every day at 3 PM.  

> The following filter enables you to retrieve all the attributes updated since the 2020-08-26 at 15:00 and matching one of the listed codes (brand, size, description, name, picture, color, secondary_color):

Reviewer: ```suggestion
The following filter enables you to retrieve all the attributes updated since August 26th, 2020 at 03:00 pm and matching one of the listed codes (brand, size, description, name, picture, color, secondary_color):
```
